frame/nicks/src/lib.rs

```diff
@@ -22,8 +22,10 @@
 //!
 //! ## Overview
 //!
-//! Nicks is a trivial module for keeping track of account names on-chain. It makes no effort to
-//! create a name hierarchy, be a DNS replacement or provide reverse lookups.
+//! Nicks is a non-production-ready module for keeping track of account names on-chain. It makes no
+//! effort to create a name hierarchy, be a DNS replacement or provide reverse lookups. Furthermore,
+//! the weights attached to this module's dispatchable functions are for demonstration purposes only
+//! and have not been designed to be economically secure.
```

Comment:
what makes it "non-production-ready", the lack of features or the incorrect weights ?

The sentence seems to imply that the lack of feature makes it non-production ready but as far as I see only the incorrect weight is an issue here no ?